projects/storefrontapp-e2e-cypress/cypress/integration/b2b/regression/saved-cart/b2b-saved-cart.e2e-spec.ts

```diff
@@ -178,6 +205,50 @@ context('B2B - Saved Cart', () => {
           sampleData.savedActiveCartForm[0]
         );
       });
+
+      it('should make saved cart active from details page', () => {
+        savedCart.waitForSavedCartListingPageData([
+          sampleData.products[0],
+          sampleData.products[1],
+          sampleData.products[2],
+        ]);
+        savedCart.visitSavedCartListingPage();
+
+        cy.get('cx-saved-cart-list .cx-saved-cart-list-table tr')
+          .first()
+          .find('.cx-saved-cart-list-cart-id > .cx-saved-cart-list-value')
+          .then((cartId) => {
+            const cartCode = cartId.text();
+
+            savedCart.openFirstElementOnTheList(cartCode);
+            savedCart.manuallyRestoryCartFromDetailsPage(cartCode);
+            savedCart.verifyMiniCartQuantity(1);
+            savedCart.visitSavedCartListingPage();
+            savedCart.verifySavedCartListQuantity(2);
+          });
+      });
+
+      it('should update saved cart product quantity, remove another product and make cart active', () => {
+        savedCart.addProductToCart(sampleData.products[0], 3);
+        savedCart.addProductToCart(sampleData.products[1], 3);
+        savedCart.saveActiveCart(true);
+        savedCart.visitSavedCartListingPage();
+
+        cy.get('cx-saved-cart-list .cx-saved-cart-list-table tr')
+          .first()
+          .find('.cx-saved-cart-list-cart-id > .cx-saved-cart-list-value')
+          .then((cartId) => {
+            const cartCode = cartId.text();
+
+            savedCart.openFirstElementOnTheList(cartCode);
+            savedCart.verifySavedCartDetailsItemsQuantity(2);
+
+            savedCart.removeFirstProductFromSavedCart(cartCode);
+            savedCart.verifySavedCartDetailsItemsQuantity(1);
+
+            savedCart.modifyFirstProductQuantityFromSaveCart(2, cartCode);
```

Comment:
I don't think you have the last 2 of what was mentioned below. I would discuss with Kate if we should really should checkout

```
-> Updating quantity in a saved cart
- Have a saved cart with a few items, for eg, 3 different items
- Go to the saved cart
- Update the quantity of one of the items
- Remove one item from the saved cart
- Make the cart active
- Proceed to checkout
```

Reply:
Asked Kate

Reply:
I have added 'make cart active' but proceed to checkout is nowhere in the saved cart e2e file so let's leave it.

Reply:
Yep, I agreed to have checkout on a separate saved cart test (written below), as long as we complete an order on a saved cart.